docs/guide/installation.md

```diff
@@ -78,11 +78,34 @@ poetry run lnbits-cli --help
 
 ```sh
 cd lnbits
-# Stop LNbits with `ctrl + x`
-git pull
-# Keep your poetry install up to date, this can be done with `poetry self update`
+# Stop LNbits with `ctrl + x` or with service manager
+# sudo systemctl stop lnbits
+
+# To keep your poetry up to date, use
+pyenv install 3.12
+pyenv local 3.12
+pyenv rehash
+poetry env use python3.12
+poetry env list
+poetry self update
```

Comment:
not sure if `poetry self update` is a good idea. we are not compatible wit hthe new poetry v2.x because of nix

Reply:
Would it be reasonable to define exact poetry version supported
with `poetry self update <version>`?

Reply:
> not sure if `poetry self update` is a good idea. we are not compatible wit hthe new poetry v2.x because of nix

removed.